src/appState.ts

```diff
@@ -116,6 +122,7 @@ const APP_STATE_STORAGE_CONF = (<
   errorMessage: { browser: false, export: false },
   exportBackground: { browser: true, export: false },
   exportEmbedScene: { browser: true, export: false },
+  exportScale: { browser: true, export: true },
```

Comment:
```suggestion
  exportScale: { browser: true, export: false },
```

`export` here indicates whether to serialize this when exporting to a file. Since this is a personal/device-specific setting, it shouldn't be persisted to a file.

Reply:
If I open a PNG with a scene in it that has scale set to 3, and then save it again, I think I would expect it to resave with the same scale.

Reply:
Yeah, I think the common case will be saving with the same scale every time on a given device. Moreover, the exported file should be device-agnostic and shouldn't overwrite user's preferences. Especially in a case of export setting, something the user may not even be aware of was changed.

(We do however already persist `gridSize` which is somewhat of a user preference as well, but the argument could be made either way — some scenes, such as architectural diagrams and grid mode can be tied together, and it's also obvious when it's enabled or not.)